src/Value/CSSString.php

```diff
@@ -83,6 +83,9 @@ public function getString(): string
         return $this->string;
     }
 
+    /**
+     * @return non-empty-string
+     */
     public function render(OutputFormat $outputFormat): string
```

Comment:
We can probably also add non-empty for `OutputFormat::$stringQuotingType` and getter/setter.